less/home-page.less

```diff
@@ -46,30 +72,92 @@
     display: flex;
     justify-content: center;
 
+    @media @phone {
+      margin-top: 0;
+      flex-direction: column-reverse;
+      padding: 15px;
+    }
+
     .walk-text {
       width: 50%;
 
+      @media @phone {
+      width: 100%;
+    }
+
       h3 {
         margin-bottom: 10px;
+
+         @media @phone {
+      margin-top: 15px;
+    }
+
       }
 
       p {
         margin-bottom: 26px;
       }
 
       .btn {
+        .main-button(@btn-bg-color, white, 4px 11px, 4px, 1.6rem);
+
+        &:hover {
+          background-color: white;
+          color: @btn-bg-color;
+        }
+
+        @media @phone {
+          .main-button(@btn-bg-color, white, 25px 42%, 4px, 1.8rem);
+        }
       }
     }
 
     .walk-img {
       width: 46%;
       margin-left: 4%;
+
+      @media @phone {
+        width: 100%;
+        margin: 0;
+      }
       
       img {
         width: 100%;
         height: auto;
         border-radius: 10px;
       }
     }
+
+  }
+
+  #contact-form {
+
+    @media @phone {
+        display: none;
+      }
+
```

Comment:
I almost missed the form because it was gone on the mobile view, lol. It was a pleasant surprise when I widened the browser